validator/src/main/resources/validations/prometheus-static-metric-validation.yml

```diff
@@ -0,0 +1,4 @@
+-
```

Comment:
do we need to retire `prometheus-sd-validation.yml` which is used to validate AMP exporter test case?

Reply:
is the current prometheus-sd-validation.yml being used? It looks the same as prometheus-metric-validation.yml.

Reply:
This validator will also be used for validating the metrics coming in from aws-otel-lambda.